.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -39,7 +41,7 @@
 
 <!-- Why should the PR reviewer trust that this change doesn't break anything? How have you tested this change? -->
 
-###### Contributor
+###### Contributor (PR submitter & devs working on this)
```

Comment:
Do we need to provide clarity in the header of this section?

Reply:
Kinda, I was confused at first, but we can drop it if you want

Reply:
Let's drop it ! ^_^